content/updates/2017-11-20-release-notes-buildpacks-sandboxes-volume-services.md

```diff
@@ -0,0 +1,54 @@
++++
+date = "2017-11-20"
+title = "cloud.gov monthly update: Buildpack notifications, expiring sandboxes and other new features"
```

Comment:
Expiring sandboxes aren't in production yet (and won't really be a feature), so I suggest replacing "expiring sandboxes" with "volume services" or another in-production feature :)